subset/bacnet/bacnetTests/src/main/java/helper/PicsValidator.java

```diff
@@ -0,0 +1,109 @@
+
+  Multimap<String, String> result = ArrayListMultimap.create();
+
+  boolean testPassed = false;
+  private int count = 0;
+
+  public void validate(String bacnetObjectType, String bacnetObjectProperty, String conformanceCode,
+                       String supported, Multimap bacnetPointsMap, boolean verboseOutput) {
+
+    Set<String> mapKeySet = bacnetPointsMap.keySet();
+    ArrayList<String> keys = getMapKeys(mapKeySet, bacnetObjectType);
+
+    if (keys.size() == 0 && (conformanceCode.contains(read) || conformanceCode.equals(write))
+            && !bacnetObjectProperty.equals("Property List")) {
+      writeToAppendix(formatProperty, bacnetObjectType, bacnetObjectProperty, conformanceCode,
+              "FAILED", verboseOutput);
+      setResult(false);
+    } else if (keys.size() == 0 && conformanceCode.contains(optional)
+            && !bacnetObjectProperty.equals("Property List")) {
+      writeToAppendix(formatProperty, bacnetObjectType, bacnetObjectProperty, conformanceCode,
+              "PASSED/WARNING", verboseOutput);
+      setResult(true);
+    } else if (keys.size() == 0 && bacnetObjectProperty.equals("Property List")) {
+      writeToAppendix(formatProperty, bacnetObjectType, bacnetObjectProperty, conformanceCode,
+              "PASSED", verboseOutput);
+      setResult(true);
+    }
+
+    for (String key : keys) {
+      String properties = bacnetPointsMap.get(key).toString();
+      boolean bacnetObjectPropertyIsFound =
+          Pattern.compile(Pattern.quote(bacnetObjectProperty), Pattern.CASE_INSENSITIVE)
+              .matcher(properties).find();
+      if (!bacnetObjectPropertyIsFound
+          && (conformanceCode.contains(read) || conformanceCode.equals(write)) && supported.equals(Supported)
+          && !bacnetObjectProperty.equals("Property List")) {
+        writeToAppendix(formatProperty, bacnetObjectType, bacnetObjectProperty, conformanceCode,
+                "FAILED", verboseOutput);
+        setResult(false);
+      } else if (!bacnetObjectPropertyIsFound && conformanceCode.contains(optional)
+          && supported.equals(Supported) && !bacnetObjectProperty.equals("Property List")) {
+        writeToAppendix(formatProperty, bacnetObjectType, bacnetObjectProperty, conformanceCode,
+                "PASSED/WARNING", verboseOutput);
+        setResult(true);
+      } else if (bacnetObjectPropertyIsFound && supported.equals(Supported)
+          && !bacnetObjectProperty.equals("Property List")) {
+        writeToAppendix(formatProperty, bacnetObjectType, bacnetObjectProperty, conformanceCode,
+                "PASSED", verboseOutput);
+        setResult(true);
+      }
+    }
+  }
+
+  private void writeToAppendix(String formatProperty, String key, String bacnetObjectProperty, String conformanceCode,
+                               String lineresult, boolean verboseOutput) {
+    if (!verboseOutput & (lineresult.contains("FAILED") | lineresult.contains("PASSED/WARNING"))) {
```

Comment:
@JoelRedstone @ewRedstone @InBrewJ the code now makes use of a boolean verboseOutput switch which is false by default and only outputs FAILED and PASSED/WARNING fields/properties.
Let's fix Travis tomorrow and then merge.